jx_base/queries.py

```diff
@@ -28,3 +28,19 @@ def is_variable_name(value):
     if not match:
         return False
     return match.group(0) == value
+
+def dequote(s):
```

Comment:
what language are you targeting here?  SQL?  Python?  JSON? In any case be sure to handle escape sequences.  If you are targeting Python, you could use `ast.literal_eval()`, or any number of other builtin parsers.